compression/compression-large-flush-output.any.js

```diff
@@ -0,0 +1,59 @@
+// META: global=window,worker
+// META: script=third_party/pako/pako_inflate.min.js
+// META: timeout=long
+
+'use strict';
+
+// This test verifies that a large flush output will not truncate the
+// final results.
+
+async function compressData(chunk, format) {
+  const cs = new CompressionStream(format);
+  const writer = cs.writable.getWriter();
+ 
+  writer.write(chunk);
+
+  const closePromise = writer.close();
+  const out = [];
+  const reader = cs.readable.getReader();
```

Comment:
Is there a reason you can't use concatenateStream from resources/concatenate-stream.js ?

Reply:
Let me a take a look. This was written outside the whole WPT framework, so there may be a few more things we could simplify here.

Reply:
Fixed